.github/workflows/ci.yml

```diff
@@ -89,6 +87,17 @@ jobs:
           # Make sure the help options print.
           python -m PyInstaller -h
 
+      - name: Install test dependencies (tools)
+        run: |
+          # Install dependencies for tests - basic tools
+          pip install --progress-bar=off -U -r tests/requirements-tools.txt
+
+      - name: Install test dependencies (libraries)
+        if: ${{ !endsWith(matrix.python-version, '-dev') }}
+        run: |
+          # Install dependencies for tests - libraries
+          pip install --progress-bar=off -U -r tests/requirements-libraries.txt
```

Comment:
Surely these don't need both a name and a comment?

Reply:
Indeed they don't.